test/kino/smart_cell/chart_builder_test.exs

```diff
@@ -43,7 +43,9 @@ defmodule Kino.SmartCell.ChartBuilderTest do
     end
 
     test "source for a basic line plot with alias" do
-      attrs = Map.merge(@defaults, %{"chart_type" => "line", "vl_alias" => Vl})
+      attrs =
+        Map.merge(@defaults, %{"chart_type" => "line", "vl_alias" => Vl})
+        |> ChartBuilder.add_layer()
```

Comment:
Instead of exposing the function we can add a helper like

```elixir
defp build_attrs(root_attrs, layer_attrs) do
  root_attrs = Map.merge(%{...}, root_attrs)
  layer_attrs = Map.merge(%{...}, layer_attrs)
  Map.put(root_attrs, "layers", [layer_attrs])
end
```

where `merge` first argument are the corresponding defaults :)